### PR TITLE
Bump to selenium 3.7.1

### PR DESCRIPTION
**Pull request by mach6**
Also bump chromedriver, iedriver, and geckodriver

- [X] By placing an `X` in the preceding checkbox, I verify that I am a PayPal employee or
I have signed the [Contributor License Agreement](https://github.com/paypal/SeLion#contributing)

---

### CODE REVIEW

**Comment by mach6** [comment]
@sebady @sundaramrajendran please review

File: server/src/main/java/com/paypal/selion/grid/AbstractBaseProcessLauncher.java [hunk 1]
```diff
@@ -133,7 +132,13 @@ private void waitForTerminationAfterDestroy(int duration, TimeUnit unit) {
         }
 
         public void destroyProcessForcefully() {
-            ProcessUtils.killProcess(process);
+            try {
+                Process awaitFor = this.process.destroyForcibly();
+                awaitFor.waitFor(10, SECONDS);
+            } catch (InterruptedException e) {
+                Thread.interrupted();
+                throw new RuntimeException(e);
```
[the remark this answers is not in the document]
Comment (sebady) [reply server/src/main/java/com/paypal/selion/grid/AbstractBaseProcessLauncher.java:140]: Should we attempt to gracefully terminate process first before `destroyForcibly`?

Also if `waitFor` throws an InterruptedException I don't think is catastrophic.  So might be better to log and ignore here.

File: server/src/main/java/com/paypal/selion/grid/servlets/SauceServlet.java [hunk 2]
```diff
@@ -186,7 +186,7 @@ private String getRegistrationRequestEntity() throws FileNotFoundException {
             gnc.maxSession = maxConcurrent;
 
             // update browser max instances for all saucelabs "browser" types
-            for (DesiredCapabilities caps : gnc.capabilities) {
+            for (MutableCapabilities caps : gnc.capabilities) {
```
Comment (sebady) [inline server/src/main/java/com/paypal/selion/grid/servlets/SauceServlet.java:189]: I guess I need read up on `MutableCapabilities`

**Review by sebady — APPROVED** [review]
Just one minor comment around `proces termination`.  The rest of this all looks good.

File: server/src/main/java/com/paypal/selion/grid/AbstractBaseProcessLauncher.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mach6) [reply server/src/main/java/com/paypal/selion/grid/AbstractBaseProcessLauncher.java:140]: the method `shutdown()` attempts graceful shutdown by calling `destroyProcess()` first and if that fails it then calls the method `destroyProcessForcefull()` which this code is in.

**Pull request merged**